autoload/leaderf.vim

```diff
@@ -293,6 +293,120 @@ let g:Lf_KeyMap = {
             \ "\<SCROLLWHEELDOWN>": '<SCROLLWHEELDOWN>'
             \}
 
+let g:Lf_DevIconsPallete = {
```

Comment:
I find the colors do not look good in dark colorscheme. Maybe they are for light colorscheme.

Reply:
Do you want to just describe the example in the doc?

Reply:
Remove the default value in `autoload/leaderf.vim` and modify the doc as follows?

```diff
  g:Lf_DevIconsPalette                            *g:Lf_DevIconsPalette*
      This is a dictionary with customizable vim-devicons highlights.
  
      Its structure is like below: >
      let g:Lf_DevIconsPallete = {
          \  '_': {
          \          'gui': 'NONE',
          \          'font': 'NONE',
          \          'guifg': 'NONE',
          \          'guibg': '#303136',
          \          'cterm': 'NONE',
          \          'ctermfg': 'NONE',
          \          'ctermbg': '236'
          \      },
          \  'default': {
          \          'guifg': '#0a0a0a',
          \      },
          \  '.gitignore': {
          \          'guifg': '#00ff00',
          \      },
          \  'vim': {
          \          'guifg': '#00e5a0',
          \      }
          \  }
  <
      Key Description:
          `'_'` :       All icons with no customized highlights.
          `'default'` : General Icons.
          Other Keys: Icons of the file type and the specified file name.
  
+     It is not defined by default.
+ 
-     Default value: >
+     Example (for light colorscheme) : >
      let g:Lf_DevIconsPallete = {
          \ '_':            { 'guifg': 'NONE',    'ctermfg': 'NONE' },
          \ 'default':      { 'guifg': 'NONE',    'ctermfg': 'NONE' },
          \ 'erl':          { 'guifg': '#a2003e', 'ctermfg': '124'  },
          \ 'styl':         { 'guifg': '#1572b6', 'ctermfg': '25'   },
          \ 'rmd':          { 'guifg': '#755838', 'ctermfg': '94'   },
          \ 'c++':          { 'guifg': '#984c93', 'ctermfg': '96'   },
          \ 'pyc':          { 'guifg': '#366994', 'ctermfg': '24'   },
          \ 'pyd':          { 'guifg': '#366994', 'ctermfg': '24'   },
          \ 'go':           { 'guifg': '#00acd7', 'ctermfg': '38'   },
          \ 'suo':          { 'guifg': '#00519a', 'ctermfg': '24'   },
          \ 'lua':          { 'guifg': '#3e6dbf', 'ctermfg': '25'   },
          \ 'bash':         { 'guifg': '#d9b400', 'ctermfg': '178'  },
          \ 'pyo':          { 'guifg': '#366994', 'ctermfg': '24'   },
          \ 'mustache':     { 'guifg': 'NONE',    'ctermfg': 'NONE' },
          \ 'gif':          { 'guifg': '#2dcc9f', 'ctermfg': '43'   },
          \ 'fs':           { 'guifg': '#378bba', 'ctermfg': '31'   },
          \ 'leex':         { 'guifg': '#6f567e', 'ctermfg': '60'   },
          \ 'lhs':          { 'guifg': '#8f4e8b', 'ctermfg': '96'   },
          \ 'csh':          { 'guifg': '#d9b400', 'ctermfg': '178'  },
          \ 'scala':        { 'guifg': '#7ce1ff', 'ctermfg': '117'  },
          \ 'zsh':          { 'guifg': '#d9b400', 'ctermfg': '178'  },
          \ 'jpeg':         { 'guifg': '#2dcc9f', 'ctermfg': '43'   },
          \ 'ts':           { 'guifg': '#007acc', 'ctermfg': '32'   },
          \ 'xcplayground': { 'guifg': '#f88535', 'ctermfg': '208'  },
          \ 'fsi':          { 'guifg': '#378bba', 'ctermfg': '31'   },
          \ 'vim':          { 'guifg': '#007F00', 'ctermfg': '28'   },
          \ 'scss':         { 'guifg': '#cd6799', 'ctermfg': '168'  },
          \ 'js':           { 'guifg': '#f5de19', 'ctermfg': '220'  },
          \ 'hbs':          { 'guifg': 'NONE',    'ctermfg': 'NONE' },
          \ 'xul':          { 'guifg': '#f1662a', 'ctermfg': '202'  },
          \ 'mdx':          { 'guifg': '#755838', 'ctermfg': '94'   },
          \ 'fish':         { 'guifg': '#d9b400', 'ctermfg': '178'  },
          \ 'java':         { 'guifg': '#5382a1', 'ctermfg': '67'   },
          \ 'htm':          { 'guifg': '#f1662a', 'ctermfg': '202'  },
          \ 'rb':           { 'guifg': '#871101', 'ctermfg': '88'   },
          \ 'f#':           { 'guifg': '#378bba', 'ctermfg': '31'   },
          \ 'c':            { 'guifg': '#005f91', 'ctermfg': '24'   },
          \ 'd':            { 'guifg': '#b03931', 'ctermfg': '124'  },
          \ 'md':           { 'guifg': '#755838', 'ctermfg': '94'   },
          \ 'h':            { 'guifg': '#984c93', 'ctermfg': '96'   },
          \ 'mli':          { 'guifg': '#cfcfcf', 'ctermfg': '251'  },
          \ 'ml':           { 'guifg': '#cfcfcf', 'ctermfg': '251'  },
          \ 'css':          { 'guifg': '#1572b6', 'ctermfg': '25'   },
          \ 'hh':           { 'guifg': '#984c93', 'ctermfg': '96'   },
          \ 'php':          { 'guifg': '#6280b6', 'ctermfg': '67'   },
          \ 'cc':           { 'guifg': '#984c93', 'ctermfg': '96'   },
          \ 'swift':        { 'guifg': '#f88535', 'ctermfg': '208'  },
          \ 't':            { 'guifg': 'NONE',    'ctermfg': 'NONE' },
          \ 'jl':           { 'guifg': '#aa79c1', 'ctermfg': '139'  },
          \ 'cljc':         { 'guifg': '#63b132', 'ctermfg': '70'   },
          \ 'exs':          { 'guifg': '#6f567e', 'ctermfg': '60'   },
          \ 'ksh':          { 'guifg': '#d9b400', 'ctermfg': '178'  },
          \ 'hs':           { 'guifg': '#8f4e8b', 'ctermfg': '96'   },
          \ 'sql':          { 'guifg': '#c4c7ce', 'ctermfg': '188'  },
          \ 'cp':           { 'guifg': '#984c93', 'ctermfg': '96'   },
          \ 'ps1':          { 'guifg': '#af4343', 'ctermfg': '124'  },
          \ 'cs':           { 'guifg': '#368832', 'ctermfg': '28'   },
          \ 'bmp':          { 'guifg': '#2dcc9f', 'ctermfg': '43'   },
          \ 'sln':          { 'guifg': '#00519a', 'ctermfg': '24'   },
          \ 'cljs':         { 'guifg': '#63b132', 'ctermfg': '70'   },
          \ 'html':         { 'guifg': '#f1662a', 'ctermfg': '202'  },
          \ 'cpp':          { 'guifg': '#984c93', 'ctermfg': '96'   },
          \ 'hrl':          { 'guifg': '#a2003e', 'ctermfg': '124'  },
          \ 'rs':           { 'guifg': '#817871', 'ctermfg': '101'  },
          \ 'sass':         { 'guifg': '#cd6799', 'ctermfg': '168'  },
          \ 'vue':          { 'guifg': '#41b883', 'ctermfg': '36'   },
          \ 'awk':          { 'guifg': '#d9b400', 'ctermfg': '178'  },
          \ 'conf':         { 'guifg': '#99b8c4', 'ctermfg': '110'  },
          \ 'pl':           { 'guifg': '#00acd7', 'ctermfg': '38'   },
          \ 'pm':           { 'guifg': '#00acd7', 'ctermfg': '38'   },
          \ 'rlib':         { 'guifg': '#817871', 'ctermfg': '101'  },
          \ 'pp':           { 'guifg': 'NONE',    'ctermfg': 'NONE' },
          \ 'dart':         { 'guifg': '#66c3fa', 'ctermfg': '81'   },
          \ 'elm':          { 'guifg': '#5fb4cb', 'ctermfg': '74'   },
          \ 'jpg':          { 'guifg': '#2dcc9f', 'ctermfg': '43'   },
          \ 'toml':         { 'guifg': '#7e7f7f', 'ctermfg': '243'  },
          \ 'py':           { 'guifg': '#366994', 'ctermfg': '24'   },
          \ 'mjs':          { 'guifg': '#f5de19', 'ctermfg': '220'  },
          \ 'less':         { 'guifg': '#2a4f84', 'ctermfg': '24'   },
          \ 'ai':           { 'guifg': '#F37021', 'ctermfg': '202'  },
          \ 'jsx':          { 'guifg': '#00d8ff', 'ctermfg': '45'   },
          \ 'webp':         { 'guifg': '#2dcc9f', 'ctermfg': '43'   },
          \ 'psd':          { 'guifg': '#26C9FF', 'ctermfg': '45'   },
          \ 'psb':          { 'guifg': '#26C9FF', 'ctermfg': '45'   },
          \ 'json':         { 'guifg': '#f5de19', 'ctermfg': '220'  },
          \ 'rss':          { 'guifg': '#00acd7', 'ctermfg': '38'   },
          \ 'cxx':          { 'guifg': '#984c93', 'ctermfg': '96'   },
          \ 'png':          { 'guifg': '#2dcc9f', 'ctermfg': '43'   },
          \ 'sh':           { 'guifg': '#d9b400', 'ctermfg': '178'  },
          \ 'hpp':          { 'guifg': '#984c93', 'ctermfg': '96'   },
          \ 'bat':          { 'guifg': '#bedcdc', 'ctermfg': '152'  },
          \ 'fsx':          { 'guifg': '#378bba', 'ctermfg': '31'   },
          \ 'ini':          { 'guifg': '#99b8c4', 'ctermfg': '110'  },
          \ 'markdown':     { 'guifg': '#755838', 'ctermfg': '94'   },
          \ 'db':           { 'guifg': '#c4c7ce', 'ctermfg': '188'  },
          \ 'twig':         { 'guifg': '#63bf6a', 'ctermfg': '71'   },
          \ 'yaml':         { 'guifg': '#ffe885', 'ctermfg': '222'  },
          \ 'coffee':       { 'guifg': '#6f4e37', 'ctermfg': '58'   },
          \ 'ejs':          { 'guifg': '#90a93a', 'ctermfg': '106'  },
          \ 'clj':          { 'guifg': '#63b132', 'ctermfg': '70'   },
          \ 'ex':           { 'guifg': '#6f567e', 'ctermfg': '60'   },
          \ 'edn':          { 'guifg': '#63b132', 'ctermfg': '70'   },
          \ 'slim':         { 'guifg': '#719E40', 'ctermfg': '70'   },
          \ 'eex':          { 'guifg': '#6f567e', 'ctermfg': '60'   },
          \ 'tsx':          { 'guifg': '#00d8ff', 'ctermfg': '45'   },
          \ 'dump':         { 'guifg': '#c4c7ce', 'ctermfg': '188'  },
          \ 'hxx':          { 'guifg': '#984c93', 'ctermfg': '96'   },
          \ 'diff':         { 'guifg': '#dd4c35', 'ctermfg': '166'  },
          \ 'ico':          { 'guifg': '#2dcc9f', 'ctermfg': '43'   },
          \ 'fsscript':     { 'guifg': '#378bba', 'ctermfg': '31'   },
          \ 'yml':          { 'guifg': '#ffe885', 'ctermfg': '222'  },
          \}
  <
      I set it with reference to https://github.com/vscode-icons/vscode-icons
```

Reply:
No, I'd like to keep it.  
I'm thinking whether to specify the colors for both dark and light colorschemes, just like https://github.com/Yggdroot/LeaderF/blob/c46ce51f500200d543953a23413400004ab45db8/doc/leaderf.txt#L786-L816

Reply:
Since `g:Lf_PopupPallete` and `g:Lf_DevIconsPallete` have similar uses, I think it would be good to be able to specify them in light and dark respectively.


May I implement it?

What do you want the default value of dark to be?

Reply:
To make it looks lighter than the current value in dark colorcheme.

Reply:
I will give it a try.

Reply:
Implemented 4458e971fe316cd97c95a61609fb5804a3c70c63.
If there is a color you don't like, please let me know.